resip/dum/SubscriptionCreator.cxx

```diff
@@ -15,6 +15,7 @@ SubscriptionCreator::SubscriptionCreator(DialogUsageManager& dum,
 
    mLastRequest->header(h_Event).value() = event; 
    mLastRequest->header(h_Expires).value() = subscriptionTime;
+   mLastRequest->header(h_Accepts).push_back( Mime( "application","pidf+xml") );
```

Comment:
This change is not appropriate - you can't add that Accept header to all subscriptions created by DUM.